### PR TITLE
Update readme to fit (re)moved CLI support

### PR DESCRIPTION
**Pull request by axe312ger**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,51 +10,142 @@
 
 [Contentful](https://www.contentful.com) provides a content infrastructure for digital teams to power content in websites, apps, and devices. Unlike a CMS, Contentful was built to integrate with the modern software stack. It offers a central hub for structured content, powerful management and delivery APIs, and a customizable web app that enable developers and content creators to ship digital products faster.
 
-This Command Line Tool (CLI) helps you to import files generated by [contentful-export](https://github.com/contentful/contentful-export) to a destination space.
+This library helps you to import files generated by [contentful-export](https://github.com/contentful/contentful-export) to a destination space.
 
-## Installation
+## :exclamation: Usage as CLI
+We moved the CLI version of this tool into our [Contentful CLI](https://github.com/contentful/contentful-cli). This allows our users to use and install only one single CLI tool to get the full Contentful experience.
```
Comment (loudmouth) [inline README.md]: same comment as my review of contentful-export. I want some more prominence of this entire sentence to make it stand out to the eye more. i suspect most usage will be via contentful-cli, so we gotta make sure people find that ;-)

File: README.md [hunk 2]
```diff
@@ -10,51 +10,142 @@
...
-let spaceImport = require('contentful-import')
-let options = {
-    content: {entries:..., contentTypes:..., locales:...},
-    spaceId: 'SPACE_ID',
-    managementToken: 'MANAGEMENT_TOKEN'
+const options = {
+  contentFile: '/path/to/result/of/contentful-export.json',
+  spaceId: '<space_id>',
+  managementToken: '<content_management_api_key>',
+  environment: '<environment-id>',
```
Comment (loudmouth) [inline README.md]: i think you need to sort out if it's `environmentId` or `environment` for both repos. and should be `<environment_id>` with and underscore 😉

File: README.md [hunk 3]
```diff
@@ -10,51 +10,142 @@
...
-## Usage and examples
+## :hand: Usage
 
-### Example
+```javascript
+const contentfulImport = require('contentful-import')
 
-#### Import an environment
+const options = {
+  content: {entries:..., contentTypes:..., locales:...},
```
Comment (Khaledgarbaya) [inline README.md]: maybe add `const contentfulImport = require('contentful-import')` to stay consistent with other snippets

File: README.md [hunk 4]
```diff
@@ -10,51 +10,142 @@
...
+
+#### Import an environment
 
 ```javascript
-let spaceImport = require('contentful-import')
-let options = {
-    content: {entries:..., contentTypes:..., locales:...},
-    spaceId: 'SPACE_ID',
-    managementToken: 'MANAGEMENT_TOKEN'
+const options = {
```
Comment (Khaledgarbaya) [inline README.md]: Same here

**Pull request merged**